pkg/controllers/kube-controller-manager_test.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright © 2022 MicroShift Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	embedded "github.com/openshift/microshift/assets"
+	"github.com/openshift/microshift/pkg/config"
+)
+
+func TestKCMDefaultConfigAsset(t *testing.T) {
+	defaultConfigBytes, err := embedded.Asset(kcmDefaultConfigAsset)
+	if err != nil {
+		t.Errorf("expected valid default config, error: %v", err)
+	}
+	if len(defaultConfigBytes) == 0 {
+		t.Errorf("expected non-empty byte array")
+	}
+}
+
+func TestConfigure(t *testing.T) {
```

Comment:
If this test is expected to fail when new configuration options appear so that we ack them and make an explicit decision about them for MicroShift, it would be good to have a comment explaining that here at the top. I don't want to hold up this work, so if add a comment like that the only reason to update this PR, it can come in another PR.